### ISSUE TITLE
Fix CSS base

### ISSUE THREAD

**Issue by oskarrough** [internet4000/radio4000#78]
Currently we rely on suitcss-base, which in turn relies on normalize.css. Either we're doing it wrong or the base is getting too opinionated. At least the suitcss-base 2->3 upgrade broke things.

Something has to do to be done. Viva!

**Comment by hugurp** [comment]
Not sure what the issue is here.
Maybe we have to remove dependencies.
Why would we need something else than `reset.css` here?

**Comment by oskarrough** [comment]
Yes. I much prefer normalize to reset because it preserves default browser styling and fixes cross-browser stuff (still very relevant).

How about we

- update and keep normalize
- remove suitcss-base and add things we need to https://github.com/internet4000/radio4000/blob/master/app/styles/base/_base.scss (pretty much just box-sizing?)
